src/apps/config/leader.lua

```diff
@@ -186,6 +186,18 @@ function Leader:rpc_set_alarm_operator_state (args)
    if success then return response else return {status=1, error=response} end
 end
 
+function Leader:rpc_purge_alarms (args)
+   local function getter()
+      if args.schema ~= self.schema_name then
+         return false, ("Purge-alarms operation not supported in"..
+                        "'%s' schema"):format(args.schema)
+      end
+      return { purged_alarms = alarms.purge_alarms(args) }
+   end
+   local success, response = pcall(getter)
+   if success then return response else return {status=1, error=response} end
```

Comment:
What's this `status=1` for?

Reply:
It's for signalling an error. I made the call using `error-reporting` as the other RPC calls in config.